docs/griptape-framework/drivers/web-search-drivers.md

```diff
@@ -82,4 +82,15 @@ Example of using `TavilyWebSearchDriver` directly:
 
 ```python
 --8<-- "docs/griptape-framework/drivers/src/web_search_drivers_4.py"
+```
+
+### Exa 
+!!! info
+    This driver requires the `drivers-web-search-exa` [extra](../index.md#extras),
+    and an Exa [api key](https://dashboard.exa.ai/api-keys)
+
+Example of using `ExaWebSearchDriver` directly:
+
+```python
+--8<-- "docs/griptape-framework/drivers/src/web_search_drivers_6.py"
```

Comment:
Is it too late to rename our web-search-driver files to `web_search_drivers/exa` instead of `web_seach_drivers_6`? Seems odd to use ints for different doc pages

Reply:
`web_search_drivers/exa.py` would be fine too. The numbers in the file names were just because I needed something deterministic for the initial generation.